includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1043,7 +1043,7 @@ function give_v18_renamed_core_settings() {
 	return [
 		'disable_paypal_verification' => 'paypal_verification',
 		'disable_css'                 => 'css',
-		'disable_welcome'             => 'welcome',
+		'disable_welcome'             => 'welcome', // @TODO: Not sure if this needs to be removed, or kept for backwards compatibility.
```

Comment:
@ravinderk Do you know of any addons that rely on/check for this setting?

Reply:
@kjohnson It is safe to remove.

Reply:
Removed.